src/main/java/org/terasology/economy/handler/CurrencyStorageHandler.java

```diff
@@ -15,13 +15,13 @@
  * Handler for currency stores.
  */
 public class CurrencyStorageHandler implements StorageComponentHandler<CurrencyStorageComponent> {
-    private final String CURRENCY = "currency";
 
-    private Logger logger = LoggerFactory.getLogger(CurrencyStorageHandler.class);
+    private static final Logger logger = LoggerFactory.getLogger(CurrencyStorageHandler.class);
+    private final String currency = "currency";
```

Comment:
Why change this this way? It looks like it was intended to be a constant value, so just making it static as well would be a better fix?
```suggestion
    private static final String CURRENCY = "currency";
```

Reply:
I was trying to resolve some of the suggestions given.